docs/elastic.rst

```diff
@@ -294,7 +294,7 @@ to allow Horovod to discover available hosts is to provide a ``--host-discovery-
 
 .. code-block:: bash
 
-    $ horovodrun -np 8 --host-discovery-script discover_hosts.sh python train.py
+    $ horovodrun --num-proc 8 --host-discovery-script discover_hosts.sh python train.py
```

Comment:
For the docs, I personally prefer keeping `-np` here, which will be very intuitive for mpi users and reinforce the fact that `horovodrun` is analogous to `mpirun`. Curious to get others' take on it.

Reply:
I kept the examples everywhere else because there was only the `-np` and not the `--min-np` / `--max-np`.

Here, in elastic, `--num-proc` comes together with `--min-num-proc` and `--max-num-proc`, and `mpirun` does not have those, so I went for this.

But I am also fine with reverting the `elastic.rst`.